File: ignored-test/java-junit/Addins_Should_Log_Warning_For_Files_Located_In_Root.java
import org.junit.jupiter.api.Test;
import org.junit.jupiter.api.Disabled;
import static org.junit.jupiter.api.Assertions.assertEquals;

import java.util.List;
import java.util.stream.Collectors;

public class Addins {

    @Test
    @Disabled("")
    public void shouldLogWarningForFilesLocatedInRoot() {
        // Given
        NuGetAddinContentResolverFixture fixture = new NuGetAddinContentResolverFixture(framework, runtime);
        fixture.createCLRAssembly("/Working/file.dll");
        fixture.createCLRAssembly("/Working/file2.dll");
        fixture.createCLRAssembly("/Working/file3.dll");

        // When
        fixture.getFiles();

        // Then
        List<LogEntry> entries = fixture.getLog().getEntries().stream()
            .filter(x -> x.getLevel() == LogLevel.WARNING &&
                         x.getMessage().equals("Could not find any assemblies compatible with " + framework + 
                         " in NuGet package " + fixture.getPackage().getPackage() + 
                         ". Falling back to using root folder of NuGet package."))
            .collect(Collectors.toList());

        assertEquals(1, entries.size());
    }
}

File: ignored-test/python-unittest/Addins_Should_Log_Warning_For_Files_Located_In_Root.py
import unittest

class TestAddins(unittest.TestCase):
    @unittest.skip("")
    def test_should_log_warning_for_files_located_in_root(self):
        fixture = NuGetAddinContentResolverFixture(framework, runtime)
        fixture.create_clr_assembly("/Working/file.dll")
        fixture.create_clr_assembly("/Working/file2.dll")
        fixture.create_clr_assembly("/Working/file3.dll")
        
        fixture.get_files()
        
        entries = [
            x for x in fixture.log.entries
            if x.level == LogLevel.WARNING and x.message == (
                f"Could not find any assemblies compatible with {framework} in NuGet package {fixture.package.package}. "
                "Falling back to using root folder of NuGet package."
            )
        ]
        
        self.assertEqual(1, len(entries))
